src/test/java/seedu/address/logic/commands/AddLessonCommandTest.java

```diff
@@ -0,0 +1,48 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandFailure;
+
+import java.time.DayOfWeek;
+import java.time.LocalTime;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.calendar.Lesson;
+
+public class AddLessonCommandTest {
+
+    private Model model;
+    private Lesson validLesson;
+
+    public void setUp() {
```

Comment:
Good abstraction :) Seems like JUnit can't really support our use case, so this is gud.

Reply:
turns out it can, mahbad, fixed!